js/foundation.abide.js

```diff
@@ -92,6 +105,35 @@ class Abide extends Plugin {
     this._init();
   }
 
+  /**
+   * Checks whether the submitted form should be validated or not, consodering formnovalidate and isEnabled
+   * @returns {Boolean}
+   * @private
+   */
+  _validationIsDisabled() {
+    if (this.isEnabled === false) { // whole validation disabled
+      return true;
+    } else if (typeof this.formnovalidate === 'boolean') { // triggered by $submit
+      return this.formnovalidate;
+    } else { // triggerd by Enter in non-submit input
```

Comment:
minor typo in comment. Should read 'triggered'.

Reply:
thanks, fixed now